ansible/roles/ocp4-workload-pipelines/tasks/remove_workload.yml

```diff
@@ -0,0 +1,69 @@
+---
+# Implement your Workload removal tasks here
+
+- name: Set up ocp4_workload_pipelines combined dictionary
+  set_fact:
+    ocp4_workload_pipelines: >-
+      {{ ocp4_workload_pipelines_defaults
+       | combine(ocp4_workload_pipelines_vars    | default( {} ),
+                 ocp4_workload_pipelines_secrets | default( {}), recursive=true )
+      }}
+
+- name: Remove Pipelines Installation
+  k8s:
+    state: absent
+    api_version: operator.tekton.dev/v1alpha1
+    kind: Config
+    name: cluster
+
+- name: Get Installed CSV
+  k8s_facts:
+    api_version: operators.coreos.com/v1alpha1
+    kind: Subscription
+    name: openshift-pipelines-operator
+    namespace: openshift-operators
+  register: r_subscription
+
+- name: Print CSV
+  debug:
+    msg: "CSV: {{ r_subscription }}"
+- name: Remove CSV
+  when:
+  - r_subscription.resources | length > 0
+  - r_subscription.resources[0].status.currentCSV is defined
+  - r_subscription.resources[0].status.currentCSV | length > 0
+  k8s:
+    state: absent
+    api_version: operators.coreos.com/v1alpha1
+    kind: ClusterServiceVersion
+    name: "{{ r_subscription.resources[0].status.currentCSV }}"
+    namespace: openshift-operators
+
+- name: Remove Subscription
+  k8s:
+    state: absent
+    definition: "{{ lookup('template', item ) | from_yaml }}"
+  loop:
+  - ./templates/subscription.j2
+
+- name: Remove openshift-pipelines project
+  k8s:
+    state: absent
+    api_version: project.openshift.io/v1
+    kind: Project
+    name: openshift-pipelines
+
+- name: Remove tkn and tkn bash completion
```

Comment:
Not that it is hurting, but is this necessary to remove?

Reply:
Well. The workload installs it. So I feel removing the workload should remove it.

Reply:
Hm. Maybe CRDs need cleaning up, too...